docs/superpowers/specs/2026-05-16-rmd-knit-preview-design.md

```
@@ -0,0 +1,454 @@
+output: html_document
+---
+
+# Hello from Raven
+
+This is a tiny R Markdown document. Run **Raven: Knit** from the command
+palette (Cmd/Ctrl+Shift+P) to render it.
+
+```{r}
+plot(1:10, main = "Example plot")
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)